src/core/connection.c

```diff
@@ -4832,7 +4832,9 @@ QuicConnRecvFrames(
             AckEliciting = TRUE;
 
             QUIC_CONNECTION_EVENT Event;
-            Event.Type = QUIC_CONNECTION_EVENT_PEER_NEEDS_STREAMS; // TODO - Uni/Bidi
+            Event.Type = QUIC_CONNECTION_EVENT_PEER_NEEDS_STREAMS;
+            Event.PEER_NEEDS_STREAMS.Bidirectional = Frame.BidirectionalStreams;
+            Event.PEER_NEEDS_STREAMS.StreamLimit = Frame.StreamLimit;
             QuicTraceLogConnVerbose(
```

Comment:
We should update this event (to `IndicatePeerNeedStreamsV2`) to reflect the values being indicated to the app.

Reply:
added 
```
Indicating QUIC_CONNECTION_EVENT_PEER_NEEDS_STREAMS type: Unidi, limit (1)
```